modules/shadertools/src/modules/phong-lighting/phong-lighting.glsl.js

```diff
@@ -43,12 +43,11 @@ vec3 lighting_getLightColor(vec3 surfaceColor, vec3 light_direction, vec3 view_d
     return (lambertian * lighting_uDiffuse * surfaceColor + specular * lighting_uSpecularColor) * intensity;
 }
 
-vec3 lighting_getLightColor(vec3 surfaceColor, vec3 position_worldspace, vec3 normal_worldspace) {
+vec3 lighting_getLightColor(vec3 surfaceColor, vec3 cameraPosition, vec3 position_worldspace, vec3 normal_worldspace) {
   vec3 lightColor = surfaceColor;
 
   if (lighting_uEnabled) {
-    vec3 camera_pos_worldspace = project_uCameraPosition;
```

Comment:
How did this uniform get in there? Double check we don't have other dependencies on deck.gl's `project` module?

Reply:
The base code is from our old lighting code, didn't notice this uniform before. No other dependencies, only this uniform now.